operate/client/src/modules/mocks/api/v2/processInstances/searchProcessInstances.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {mockPostRequest} from '../../mockRequest.ts';
```

Comment:
Let's import from the `modules/...` too

```suggestion
import {mockPostRequest} from '../../mockRequest';
```